README.md

```diff
@@ -98,6 +98,37 @@ variables for the test scripts like so:
 
 > `$ make DOCKER_OPTS="-e TEST_ALL_RUBIES=1" test`
 
+In order to run the tests manually in the container-based development, the services
+needed to run them correctly must be started before with:
+
+> `$ script/services start`
+
+Then you can execute the following commands to execute all tests:
+
+> `$ bundle exec rake`
+
+Another alternative to execute all tests from within the development environment
+where it is not needed to manually start the services before is:
+
+> `$ script/test`
+
+You can also specify execution of individual tests or type of tests.
+
+If the test is a Unit::Test type test (located in test directory) you can execute:
+> `$ bundle exec rake test:[unit|integration|special] [TEST=<test-file]>`
+
+If the test is a RSpec type test (located in spec directory) you can execute:
+> `$ bundle exec rake spec:[unit|integration|acceptance|api|use_cases] SPEC=<test-file>`
+
+Alternatively there is another syntax to execute a RSpec type test:
+> `$ bundle exec rake spec:specific[<test-file-location>]`
+
+where in this last case the [ ] characters must be literally placed.
+
+For more information on accepted commands for testing with Rake you can execute:
```

Comment:
you could also document how to use rspec directly with `bundle exec rspec <parameters> <spec_file>` to run a specs file.